### ISSUE TITLE
Validation accuracy is very low - MxNet/Classification/RN50v1.5 

### ISSUE THREAD

**Issue by beanliao** [NVIDIA/DeepLearningExamples#144]
Hi,  I used 8xV100 to run  MxNet/Classification/RN50v1.5.
The validation accuracy is very low after finished  90 epochs.  
(Validation-accuracy=0.000880, Train-accuracy=0.796514)
Is is the sample script issue?  or something wrong with my configuration?

**Config:**
1. MxNet Docker : 19.07-py3
2. Driver: 418.67
3. Dataset imagenet 2012  

**Log:**
[mxnet_rn50_fp16.log](https://github.com/NVIDIA/DeepLearningExamples/files/3460164/mxnet_rn50_fp16.log)

**Running command:**
./RN50_FP16_8GPU.sh /workspace/examples/resnet50v1.5 --data-root /datasets/imagenet_raw

**Comment by maciej-sypetkowski** [comment]
There is a mistake in the README in dataset preparation section. `im2rec.py` tool requires dataset in slightly different format then raw imagenet to convert into RecordIO format correctly.
More precisely, raw imagenet validation images are in `val/` directory and `im2rec.py` requires them to be in `val/<class identifier>/` directories.
To convert raw imagenet into this format you can run `cd <path to your raw imagenet>/val && wget -qO- https://raw.githubusercontent.com/soumith/imagenetloader.torch/master/valprep.sh | bash` and then use commands from the README to generate dataset in RecordIO format. Note that training dataset is already in good format, so there's no need to generate it again.

`val.lst` file should look like this:
```11251   225.000000      n02105162/ILSVRC2012_val_00004754.JPEG
5926    118.000000      n01978287/ILSVRC2012_val_00023583.JPEG
43812   876.000000      n04493381/ILSVRC2012_val_00015609.JPEG
19737   394.000000      n02640242/ILSVRC2012_val_00036026.JPEG
...
```
Second column is the class of given image (I guess in your case there are only zeros in this column, and that's the reason of low validation accuracy).

**Comment by beanliao** [comment]
Thanks,it was the dataset issue, I can get expected validation accuracy after following your step to prepare Val dataset.
